### DISCUSSION TITLE
跪求解答：如何将自己训练好的模型打成whl文件，目前打的包没有包含我的，需在哪里修改吗？

### DISCUSSION THREAD

**Discussion by 694578908** [PaddlePaddle/PaddleOCR#13411] [Q&A]
![image](https://github.com/user-attachments/assets/330323a0-1dac-4415-aa70-8bf446f62ecf)

**Comment by GreatV** [comment]
可以看看这些文档：
https://setuptools.pypa.io/en/latest/userguide/quickstart.html
https://wheel.readthedocs.io/en/stable/user_guide.html

**Reply by 694578908** [reply]
是在这里面加参数吗？
![image](https://github.com/user-attachments/assets/1834b68b-b865-48c1-a9a2-660809bee109)

**Reply by SWHL** [reply]
是的，多看看官方文档，摸索一下就知道了
